nodeup/pkg/model/kubelet.go

```diff
@@ -32,6 +33,10 @@ import (
 
 	v1 "k8s.io/api/core/v1"
 	"k8s.io/klog/v2"
+	kubelet "k8s.io/kubelet/config/v1beta1"
```

Comment:
What we have to avoid is using the types from componentconfigs into our own API structs. But here we are just using the structs for generating the yaml file. The alternative would be to copy the structs into our tree, which I do not think is necessary unless upstream decides to remove `v1beta1`. But right now, they even have `v1alpha1` in their tree.

Reply:
👍